### ISSUE TITLE
Add CSS files in iruby notebook

### ISSUE THREAD

**Issue by Prakriti-nith** [SciRuby/iruby#146]
I am facing a problem while adding CSS in IRuby notebook. Is there any method (like IRuby.javascript() to add js) to include CSS in IRuby notebook?

**Comment by mrkn** [comment]
@Prakriti-nith Could you try Jupyter notebook's custom.css?

**Comment by Prakriti-nith** [comment]
Yes, that will help but I need to load the CSS files from code. I am working on daru-view for which I have to load the dependent CSS file in IRuby notebook whenever user sets the plotting library. Any help would do.

**Comment by kylekyle** [comment]
Would this work?

```ruby
IRuby.display(IRuby.html(
          Erector.inline{ style raw(widget_css) }.to_html
        ))
```
